overview/content-types/demo-instructions.nix

```diff
@@ -36,11 +36,21 @@ in
           ];
         }
         {
-          platform = "Ubuntu";
+          platform = "Arch";
           shell-session.bash = [
             {
               input = ''
-                apt install --yes curl git jq nix
+                pacman --sync --refresh --noconfirm curl git jq nix
+              '';
+            }
+          ];
+        }
+        {
+          platform = "Other";
+          shell-session.bash = [
+            {
+              input = ''
+                sh <(curl --proto '=https' --tlsv1.2 -L https://nixos.org/nix/install)
               '';
```

Comment:
We discussed using the installer before and decided that using the packaged Nix would be a better alternative, so I'm not sure if we'd like to add this. How about we keep the current platforms and discuss this again, later?

Reply:
If we include this method of installation, we'd have to support it to some degree, which might prove to be too much for us at this moment. We should also consider users, who'd not only have to trust this method, but who won't have a clue of how to uninstall either.

We can definitely discuss whether or not we'd like to add or support the installer and other platforms, but I feel like it would be better to do it outside this PR, which is kinda out of its scope.

Reply:
How about instead, something like "See https://nixos.org/download/". The "Other" tab only affects people who aren't supported by the first tabs, so it would just make things easier for them by linking to the page they would get to if they feel bothered to try and figure out how to install Nix on their (not covered by the first tabs) system.

Reply:
Sure, please open a new PR and we can discuss this properly, there.